spec/features/consumer/cookies_spec.rb

```diff
@@ -0,0 +1,104 @@
+require 'spec_helper'
+
+feature "Cookies", js: true do
+  include CacheHelper
+
+  describe "banner", js: true do
+    describe "in the homepage" do
+      before do
+        Spree::Config[:cookies_consent_banner_toggle] = true
+        visit root_path       
+      end
+
+      scenario "does not show after cookies are accepted" do
+        click_button I18n.t('legal.cookies_banner.cookies_accept_button')
+        expect_not_visible_cookies_banner
+
+        visit root_path
+        expect_not_visible_cookies_banner
+        delete_spree_config_from_cache :cookies_consent_banner_toggle
```

Comment:
Now you do this multiple times. Once in `after` should be enough.

Did you actually test if you need this? I just tried it without resetting in one spec and it seems like the preference is not persisted between the scenarios. So it looks like I have been wrong and you don't need to delete it from the cache. I'm not sure why. It has been a problem in other specs and that's why there is a reset in the spec helper: https://github.com/openfoodfoundation/openfoodnetwork/blob/32846af23019e4f00d0f7b5fdb1d9fb797641912/spec/spec_helper.rb#L108-L118

So I would say, let's try it without the delete again and see if it causes any problems. Or did you verify that already?

Reply:
yes, there was an unnecessary duplication of the delete.

I thought I have tested it! But well, I have retested it and the configs are all reset for each test without the delete. So, in this particular case, the delete cache is not necessary. I have removed the cache_helper and the delete calls from this test.